tests/test_openmxparser.py

```diff
@@ -294,3 +295,36 @@ def test_graphite(parser):
     assert method.electronic.van_der_waals_method == 'G10'
     assert method.scf.n_max_iteration == 100
     assert method.scf.threshold_energy_change.magnitude == approx(Ha_to_J(1e-8))
+
+    calculation = run.calculation[0]
+    stress_tensor = calculation.stress.total.value
+    stress_tensor_expected = (
+        np.array(
+            [
+                [-0.00001778, -0.00000244, -0.00000000],
+                [-0.00000244, -0.00001879, 0.00000000],
+                [-0.00000000, 0.00000000, -0.00002298],
+            ]
+        )
+        * units.hartree
+        / units.bohr**3
+    )
+    assert np.allclose(stress_tensor, stress_tensor_expected, 0.0)
+
```

Comment:
Please also test the values from the last MD step (i.e., `calculation[-1]`)  just to be sure we get them all?